tests/css/CSSCombBearTest.py

```diff
@@ -0,0 +1,242 @@
+from bears.css.CSSCombBear import CSSCombBear
+from tests.LocalBearTestHelper import verify_local_bear
+import os
+
+test_file1 = """
+example {
+    color: red;
+}
+"""
+test_file_space_before_colon = """
+example {
+    color: red;
+}
+"""
+test_file_multiple = """
+example {
+    content: "secure"
+    background-color: lightblue
+}
+"""
+test_file_multiple2 = """
+example {
+    content: 'secure';
+    background-color: lightblue;
+}
+"""
+test_file_space_before_colon2 = """
+example {
+    color : red;
+}
+"""
+test_file2 = """
+example {color: red}
+"""
+
+test_file3 = """
+example {
+color: red;
+background-color: lightblue}
+"""
+
+test_file4 = """
+example {
+color: red;
+margin-bottom: 100px;
+margin-right: 150px;
+margin-left: 80px}
+"""
+
+test_file5 = """
+example{
+    color: red;
+    margin-bottom: 100px;
+    margin-right: 150px;
+}
+"""
+test_file6 = """
+example{color :red;
+}
+"""
+test_file_space_after_colon = """
+a {
+    color: red;
+}
+"""
+test_file_always_semicolon = """
+example{
+    color:red
+}
+"""
+
+test_file_space_after = """
+example {
+    margin-bottom: 100px;
+}
+"""
+
+test_file_space_after_close_brace = """
+example {
+    margin-bottom: 100px;
+}
+"""
+
+test_file_remove_empty_rulesets = """
+example {
+    margin-bottom: 100px;
+}
+example2 {}
+"""
+
+test_file_add_semicolon = """
+example {
+    margin-bottom: 200px
+}
+"""
+
+test_file_color_case = """
+example {
+    color: #fff;
+}
+"""
+
+test_file_color_case2 = """
+example {
+    color: #FFF;
+}
+"""
+
+test_file_block_indent = """
+example {
+color: tomato;
+}
+"""
+
+test_file_block_indent2 = """
+example {
+    color:tomato;
+}
+"""
+
+csscombconfig = os.path.join(os.path.dirname(__file__),
+                             "test_files",
+                             "csscomb.json")
+
+settings = {
+    "always-semicolon": "True",
+    "color-case": "upper",
+    "eof-newline": "True",
```

Comment:
what you have in your bear is ``eof_newline`` not ``eof-newline``
and the same goes for all the other settings (note what you use as setting in you .coafile is what is available to the bear)